R/shared-methods.R

```diff
@@ -175,10 +116,65 @@ DataClass <- R6::R6Class(
     set_region_codes = function() {
     },
 
-    #' @description Initialize the country
-    #' @param ... Parameters passed to `initialise_dataclass`.
-    initialize = function(...) {
-      initialise_dataclass(self, ...)
+    #' @description initialize function used by all `Country` class objects.
+    #' Set up the country class with attributes set to input
+    #' parameters. Should only be called by a `Country` class object.
+    #' @param level A character string indicating the target administrative
+    #' level of the data with the default being "1". Currently supported
+    #' options are level 1 ("1) and level 2 ("2").
+    #' Use [get_available_datasets] for supported  options by dataset.
```

Comment:
again not sure this will link... `get_available_datasets()` may do it or markdown linking. Should be able to check it when you render docs locally.

Reply:
`\code{\linnk{function_name}}` seems to work. Will replace affected files here.